packages/textfield/src/textfield.ts

```diff
@@ -33,6 +33,70 @@ import { nothing } from 'lit-html';
  * @slot icon - The icon that appears on the left of the label
  */
 
+type AutoComplete =
+    | 'on'
+    | 'off'
+    | 'additional-name'
+    | 'address-level1'
+    | 'address-level2'
+    | 'address-level3'
+    | 'address-level4'
+    | 'address-line1'
+    | 'address-line2'
+    | 'address-line3'
+    | 'bday'
+    | 'bday-year'
+    | 'bday-day'
+    | 'bday-month'
+    | 'billing'
+    | 'cc-additional-name'
+    | 'cc-csc'
+    | 'cc-exp'
+    | 'cc-exp-month'
+    | 'cc-exp-year'
+    | 'cc-family-name'
+    | 'cc-given-name'
+    | 'cc-name'
+    | 'cc-number'
+    | 'cc-type'
+    | 'country'
+    | 'country-name'
+    | 'current-password'
+    | 'email'
+    | 'family-name'
+    | 'fax'
+    | 'given-name'
+    | 'home'
+    | 'honorific-prefix'
+    | 'honorific-suffix'
+    | 'impp'
+    | 'language'
+    | 'mobile'
+    | 'name'
+    | 'new-password'
+    | 'nickname'
+    | 'organization'
+    | 'organization-title'
+    | 'pager'
+    | 'photo'
+    | 'postal-code'
+    | 'sex'
+    | 'shipping'
+    | 'street-address'
+    | 'tel-area-code'
+    | 'tel'
+    | 'tel-country-code'
+    | 'tel-extension'
+    | 'tel-local'
+    | 'tel-local-prefix'
+    | 'tel-local-suffix'
+    | 'tel-national'
+    | 'transaction-amount'
+    | 'transaction-currency'
+    | 'url'
+    | 'username'
+    | 'work';
```

Comment:
This is kind of big. Prettier insists on one item per line 😛

Reply:
I don't think this is complete. autocomplete can include multiple tokens - for example `autocomplete="section-blue shipping street-address"` is a valid autocomplete value (taken from HTML spec)

Reply:
Ah, good catch. That wasn't clear from the MDN docs. I will resort to using a plain string. I was working off of the Typescript typings, which may have made the same mistake.